File: 2908-minimum-sum-of-mountain-triplets-i/2908-minimum-sum-of-mountain-triplets-i.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumSum = function(nums) {
    let sum=Infinity;
    for(let i=0;i<nums.length;i++){
        for(let j=i+1;j<nums.length;j++){
            for(let k=j+1;k<nums.length;k++){
                let currentsum=0;
                if(nums[i]<nums[j] && nums[k]<nums[j]){
                   currentsum=nums[i]+nums[j]+nums[k]
                   if(currentsum <sum) sum=currentsum;
                }
            }
        }
    }
    return sum === Infinity ? -1 : sum;
};

File: 2908-minimum-sum-of-mountain-triplets-i/2908-minimum-sum-of-mountain-triplets-i.cs
public class Solution {
    public int MinimumSum(int[] nums) {
        int sum = int.MaxValue;
    for (int i = 0; i < nums.Length; i++) {
        for (int j = i + 1; j < nums.Length; j++) {
            for (int k = j + 1; k < nums.Length; k++) {
                int currentsum = 0;
                if (nums[i] < nums[j] && nums[k] < nums[j]) {
                    currentsum = nums[i] + nums[j] + nums[k];
                    if (currentsum < sum) sum = currentsum;
                }
            }
        }
    }
    return sum == int.MaxValue ? -1 : sum;
    }
}